backend/account/authentication_controller.py

```diff
@@ -259,6 +260,8 @@ def make_user_organization_display_name(self, user_name: str) -> str:
         return self.auth_service.make_user_organization_display_name(user_name)
 
     def user_logout(self, request: Request) -> Response:
+        session_id: str = request.COOKIES.get("sessionid")
```

Comment:
@jagadeeswaran-zipstack I don't think cookie is used for session_id anymore, can you confirm with @muhammad-ali-e 
You can refer `UserSessionUtils` class in user_session.py. Maybe add a new static_method if required.
cc: @johnyrahul @chandrasekharan-zipstack

Reply:
There is no method available to get session id. sure I'll check with @muhammad-ali-e and add one